### PR TITLE
brainbuilder@0.13.0

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ packages:
     specs:
       - archngv@0.0.0
       - asciitoh5@1.0
-      - brainbuilder@0.12.1
+      - brainbuilder@0.13.0
```
Comment (tomdele) [inline deploy/packages/bbp-packages.yaml:26]: when we do this, is 0.12.1 still in unstable ?

Comment (asanin-epfl) [reply deploy/packages/bbp-packages.yaml:26]: I don't know to be honest but `0.12.0` and `0.11.7` are included in `archive/2020-06`. May be `0.12.1` and `0.13.0` will be included both in `archive/2020-07`?
```
module load archive/2020-06
-bash-4.2$ module avail |& grep brainbuilder
brainbuilder/0.11.7                 py-bglibpy/4.2.23
brainbuilder/0.12.0                 py-bluepy/0.14.6
```

Comment (tomdele) [reply deploy/packages/bbp-packages.yaml:26]: We will need to check for the snakemake then

**Comment by tomdele** [comment]
good on my side. 
@matz-e can we validate simple PRs like this one without your approval ?

**Comment by asanin-epfl** [comment]
even if Matthias approves, I am not going to merge this on Friday evening, no-no

File: deploy/packages/bbp-packages.yaml (hunk 1 above)
Comment (matz-e) [reply deploy/packages/bbp-packages.yaml:26]: > when we do this, is 0.12.1 still in unstable ?

Yes. Software does not get deleted… they will just drop out of "scope".

**Review by matz-e — APPROVED** [review]

**Comment by matz-e** [comment]
> good on my side.
> @matz-e can we validate simple PRs like this one without your approval ?

@genric and @wizmer should be able to approve, too.

**Comment by tomdele** [comment]
>@genric and @wizmer should be able to approve, too.

Ok ! Thanks.

**Pull request merged**